tests/fuzztests/fuzztests.cpp

```diff
@@ -29,7 +29,7 @@
 #include <windows.h>
 #endif
 
-#include "../../concurrentqueue.h"
```

Comment:
these changes required to conform to the <libname>/<include-file> syntax of other std libraries released for public and installed under <prefix>/<libname>/...